views/attendance/attendance_edit_class_info.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * enter information to take attendance for new class
+ *
+ * provides a list of fields to specify what class attendance will be recorded for
+ *
+ * @author Scott Hansen
+ * @copyright 2017 Marist College
+ * @version [version number]
+ * @since [initial version number]
+ */
+
+authorizedPage();
+
+global $db;
+
+include ('attendance_utilities.php');
+
+//form options
+$result_curriculum = $db->no_param_query("SELECT c.curriculumid, c.curriculumname FROM curricula c ORDER BY c.curriculumname ASC;");
+
+$result_classes = $db->no_param_query("SELECT cc.curriculumid, topicname, cc.classid FROM curriculumclasses cc, classes WHERE classes.classid = cc.classid ORDER BY cc.curriculumid;");
+
+$result_sites = $db->no_param_query("SELECT s.sitename FROM sites s;");
+
+$result_languages = $db->no_param_query("SELECT * FROM languages;");
+
+$result_facilitators = $db->no_param_query("SELECT peop.firstname, peop.middleinit, peop.lastname, peop.peopleid " .
+    "FROM people peop, employees emp, facilitators f " .
+    "WHERE peop.peopleid = emp.employeeid " .
+    "AND emp.employeeid = f.facilitatorid " .
+    "AND f.df = 0 " .
```

Comment:
With the updated database this should be `f.df IS FALSE`

Reply:
@blackjack26 what DB has the update? I'm connected to 10.11.12.34 and is false isn't working